app.nestful.Nestful.yml

```diff
@@ -0,0 +1,35 @@
+id: app.nestful.Nestful
+
+runtime: org.gnome.Platform
+runtime-version: '47'
```

Comment:
do you really need the GNOME runtime?

Reply:
Yes, and specifically 47 -- earlier versions did not function correctly